darts/utils/statistics.py

```diff
@@ -170,6 +173,11 @@ def extract_trend_and_seasonality(
         "The model must be either MULTIPLICATIVE or ADDITIVE.",
     )
 
+    raise_if_not(
+        (method == "MSTL") & (isinstance(freq, list)) or (method != "MSTL"),
+        f"MSTL method requires a list of frequencies. Got a single value instead: {freq}.",
+    )
```

Comment:
Sorry, my previous comment was not specific enough, what I meant was if the method is not "MSTL", we show an elegant message saying that the method (`f"... {method} ..."`) only support integer.